src/transformers/models/gpt_sw3/tokenization_gpt_sw3.py

```diff
@@ -300,15 +327,38 @@ def encode_fast(
         return token_ids
 
     def decode_fast(self, token_ids: Union[int, List[int]]) -> str:
-        """
-        Encodes a text or batch of texts to token ids using preprocessing and the raw SP tokenizer. This has reduced
-        functionality but is often much faster.
+        """Decodes token IDs to text.
+
+        This uses preprocessing and the raw SP tokenizer. This has reduced functionality but is often much faster.
 
         Args:
-            token_ids (`int` or `List[int]`): Encoded token or text as token id(s).
+            token_ids (`int` or `List[int]`):
+                Encoded token or text as token id(s).
 
         Returns:
-            `str`: Decoded text
+            `str`:
+                Decoded text.
         """
 
         return self.sp_model.decode(token_ids)
+
+    def _build_conversation_input_ids(self, conversation: Conversation) -> List[int]:
```

Comment:
```suggestion
    def _build_conversation_input_ids(self, conversation: "Conversation") -> List[int]:
```
as we don't want to have to import the `Conversation` pipelilne, we use `TYPE_HINT`